specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,575 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-12-01",
+    "title": "WebApplicationFirewallManagement",
+    "description": "APIs to manage web application firewall rules"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies": {
+      "get": {
+        "tags": [
+          "ListWebApplicationFirewallPolicies"
+        ],
+        "description": "Lists all of the protection policies within a resource group.",
+        "operationId": "Policies_List",
+        "x-ms-examples": {
+          "Lists all WAF policies in a resource group": {
+            "$ref": "./examples/WafListPolicies.json"
+          }
+        },
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/WebApplicationFirewallPolicyListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/WebApplicationFirewallErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/ApplicationGatewayWebApplicationFirewallPolicies": {
+      "get": {
+        "tags": [
+          "ListWebApplicationFirewallPolicies"
+        ],
+        "operationId": "WebApplicationFirewallPolicyListAll",
```

Comment:
Convention for naming `operationId` is "Plural form of resource type" + underscore + "Verb" (e.g. `WebApplicationFirewallPolicies_List`). Please make `WebApplicationFirewallPolicy` plural and add `_`